packages/components/src/utils/test/events.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import { mergeEventHandlers } from '../events';
+
+describe( 'mergeEventHandlers', () => {
```

Comment:
Assuming that we'd expect _all_ values from the second argument to get merged into `mergeEventHandlers`'s return value, should we add a unit test which covers this case?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/33205/commits/a41f480751de0bb41a9f186266c90435817f706d